router.go

```diff
@@ -601,6 +625,21 @@ func getVersion(c echo.Context) error {
 	doc.Rev = ""
 	doc.Attachments = nil
 
+	// Check if we have the local tarfile
+
+	conf, err := config.GetConfig()
+	if err != nil {
+		return err
+	}
+	sc := conf.SwiftConnection
+
+	base := filepath.Base(doc.URL)
+	fp := filepath.Join(appSlug, version, base)
+	buf := new(bytes.Buffer)
+	if _, err := sc.ObjectGet(prefix, fp, buf, false, nil); err == nil {
```

Comment:
You are leaking file descriptors here: you should use the response to call `Close` on it, or better make a `HEAD` request instead of `GET`